<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>historyItemWidget</class>
 <widget class="QFrame" name="historyItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>612</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="German" country="Germany"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="frame1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_date">
      <property name="text">
       <string>date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_type">
      <property name="text">
       <string>type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_status">
      <property name="text">
       <string>status</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_recipient">
      <property name="text">
       <string>recipient</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_value">
      <property name="text">
       <string>value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
